ironfish/src/rpc/routes/router.ts

```diff
@@ -52,42 +52,18 @@ export function parseRoute(
 }
 
 export class Router {
-  routes = new Map<string, Map<string, { handler: RouteHandler; schema: YupSchema }>>()
-  context: RequestContext = {}
-  server: RpcServer | null = null
-
-  register<TRequestSchema extends YupSchema, TResponse>(
-    route: string,
-    requestSchema: TRequestSchema,
-    handler: RouteHandler<YupSchemaResult<TRequestSchema>, TResponse>,
-  ): void {
-    const [namespace, method] = parseRoute(route)
-
-    Assert.isNotUndefined(namespace, `Invalid namespace: ${String(namespace)}: ${route}`)
-    Assert.isNotUndefined(method, `Invalid method: ${String(namespace)}: ${route}`)
-
-    let namespaceRoutes = this.routes.get(namespace)
-
-    if (!namespaceRoutes) {
-      namespaceRoutes = new Map<string, { handler: RouteHandler; schema: YupSchema }>()
-      this.routes.set(namespace, namespaceRoutes)
-    }
+  routes = new Routes()
```

Comment:
this gets over-ridden in the constructor you can avoid initializing this and it works right

Reply:
Yea good call